### PR TITLE
Feature/upgrade

### PR DESCRIPTION
**Pull request by aarti-joshi-nttd**

### CODE REVIEW

File: .lcafenv [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Use this file to preset variables used by the Makefile.
-# This file will be included when make is run. The variables below will
+# This file will be included when make is run. The variables below, will
 # take precedence over what is defined in Makefile when they are set. This
 # allows overriding certain settings without modifying the Makefile.
 
 # REPO_MANIFESTS_URL="https://github.com/launchbynttdata/launch-common-automation-framework.git"
-# REPO_BRANCH="refs/tags/1.0.0"
+# REPO_BRANCH="main"
```
Comment (chris11-taylor-nttd) [inline .lcafenv:7]: This shouldn't have updated. Can you undo this whole file? We should be referencing `refs/tags/1.0.0` here, not a branch name.

File: NOTICE [hunk 2]
```diff
@@ -4,4 +4,4 @@ Software, source files, and snippets are licensed under the Apache 2.0 license.
 
 All other content (e.g. documentation), unless otherwise specified, is licensed under the Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International (CC BY-NC-ND 4.0) License (<https://creativecommons.org/licenses/by-nc-nd/4.0/>).
 
-If you discover a file that does not specify its license that should be licensed, please contact the repository owners as soon as possible to resolve the issue.
+If you discover a file that does not specify specify its license that should be licensed, please contact the repository owners as soon as possible to resolve the issue.
```
Comment (chris11-taylor-nttd) [inline NOTICE]: This shouldn't have updated. Can you undo this whole file?

File: versions.tf [hunk 3]
```diff
@@ -11,16 +11,12 @@
 // limitations under the License.
 
 terraform {
-  required_version = ">= 1.5.0, <= 1.5.5"
+  required_version = "<= 1.5.5"
```
Comment (chris11-taylor-nttd) [inline versions.tf]: I don't see a reason to remove this constraint, let's revert this line.

File: .lcafenv (hunk 1 above)
Comment (aarti-joshi-nttd) [reply .lcafenv:7]: looks like we need to make some changes in this - https://github.com/launchbynttdata/lcaf-skeleton-terraform/blob/main/.lcafenv

File: NOTICE (hunk 2 above)
Comment (aarti-joshi-nttd) [reply NOTICE]: same as above

File: versions.tf (hunk 3 above)
Comment (aarti-joshi-nttd) [reply versions.tf]: fixed

**Review by chris11-taylor-nttd — APPROVED** [review]

**Review by ben-vaughan-nttd — APPROVED** [review]

**Pull request merged**